vault/resource_mfa_okta_test.go

```diff
@@ -0,0 +1,50 @@
+package vault
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-vault/testutil"
+)
+
+func TestMFAOktaBasic(t *testing.T) {
+	mfaOktaPath := acctest.RandomWithPrefix("mfa-okta")
```

Comment:
Shadow declaration of `mfaOktaPath()`.

It's preferable to use generic variable names in this scope.
```suggestion
	path := acctest.RandomWithPrefix("mfa-okta")
```

Reply:
Fixed in 5d506d7